packages/flutter_tools/lib/src/commands/analyze.dart

```diff
@@ -23,11 +25,13 @@ class AnalyzeCommand extends FlutterCommand {
     required Logger logger,
     required ProcessManager processManager,
     required Artifacts artifacts,
+    List<ProjectValidator>? allProjectValidators,
```

Comment:
I suggest making this a required argument once we have the enough useful project validators and update `executable.dart` to pass all the validators. Let me know what you think @christopherfujino

Reply:
Why not just make it required now and pass an empty list?

Reply:
It will be the same just this PR will get more modified files (between tests and places where analyze is used) I am fine with both approaches